frontend/app/components/transportation/tdf/modal-splits/census-tracts-map.js

```diff
@@ -80,6 +86,20 @@ export default class TransportationCensusTractsMapComponent extends Component {
   mapLoaded(map) {
     map.on('data', onMapStyleLoaded);
 
+    map.addControl(new MapboxAccessibility({
+      // A string value representing a property key in the data. This
+      // will be used as the text in voiceover.
+      accessibleLabelProperty: 'name',
+
+      // The layers within the style that
+      // 1. Contain the `accessibleLabelProperty` value as a key
+      // 2. Should be used for voiceover.
+      // 3. Will be represented in the DOM with nodes
+      layers: [
+        'bbls',
```

Comment:
This class will have to know about the layers that will be rendered in the template which is a weakness because this could fall out of sync with things that are rendered